<ui version="4.0" >
 <class>pqCreateServerStartupDialog</class>
 <widget class="QDialog" name="pqCreateServerStartupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure New Server</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="4" column="0" >
      <widget class="QLabel" name="dataServerHostLabel" >
       <property name="text" >
        <string>Data Server host</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Server Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="type" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>Client / Server</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Client / Server (reverse connection)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Client / Data Server / Render Server</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Client / Data Server / Render Server (reverse connection)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="host" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="hostLabel" >
       <property name="text" >
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="dataServerHost" />
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="renderServerHostLabel" >
       <property name="text" >
        <string>Render Server host</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="renderServerHost" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="name" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="portLabel" >
       <property name="text" >
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="port" >
       <property name="maximum" >
        <number>999999999</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>11111</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="dataServerPortLabel" >
       <property name="text" >
        <string>Data Server port</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QSpinBox" name="dataServerPort" >
       <property name="maximum" >
        <number>999999999</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>11111</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="renderServerPortLabel" >
       <property name="text" >
        <string>Render Server port</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QSpinBox" name="renderServerPort" >
       <property name="maximum" >
        <number>999999999</number>
       </property>
       <property name="value" >
        <number>22222</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Configure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>509</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>type</tabstop>
  <tabstop>host</tabstop>
  <tabstop>port</tabstop>
  <tabstop>dataServerHost</tabstop>
  <tabstop>dataServerPort</tabstop>
  <tabstop>renderServerHost</tabstop>
  <tabstop>renderServerPort</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>pqCreateServerStartupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>442</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>92</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>pqCreateServerStartupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>525</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>175</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
